src/components/audio-kit/audioPlayer.tsx

```diff
@@ -172,3 +198,4 @@ export const AudioPlayer: React.FC<Props> = ({
     </div>
   );
 };
+export default AudioPlayer;
```

Comment:
Let's use one style for exporting components:

`export const AudioPlayer: React.FC<Props> = ({`